cdap-client-tests/src/test/java/co/cask/cdap/client/ProgramClientTestRun.java

```diff
@@ -116,4 +128,26 @@ public void testAll() throws Exception {
     assertProgramStopped(programClient, FakeApp.NAME, ProgramType.FLOW, FakeFlow.NAME);
 
   }
+
+  @Test
+  public void testStreamWrite() throws Exception {
```

Comment:
It seems like the test cases in .*ClientTestRun should focus on testing the clients, and not to test new features in cdap.